pkg/proxy/ipvs/ipset.go

```diff
@@ -162,18 +162,18 @@ func (set *IPSet) syncIPSetEntries() {
 		for _, entry := range currentIPSetEntries.Difference(set.activeEntries).List() {
 			if err := set.handle.DelEntry(entry, set.Name); err != nil {
 				if !utilipset.IsNotFoundError(err) {
-					klog.Errorf("Failed to delete ip set entry: %s from ip set: %s, error: %v", entry, set.Name, err)
+					klog.ErrorS(err, "Failed to delete ip set entry from ip set", "entry", entry, "ipSet", set.Name)
 				}
 			} else {
-				klog.V(3).Infof("Successfully delete legacy ip set entry: %s from ip set: %s", entry, set.Name)
+				klog.V(3).InfoS("Successfully delete legacy ip set entry from ip set", "entry", entry, "ipSet", set.Name)
 			}
 		}
 		// Create active entries
 		for _, entry := range set.activeEntries.Difference(currentIPSetEntries).List() {
 			if err := set.handle.AddEntry(entry, &set.IPSet, true); err != nil {
-				klog.Errorf("Failed to add entry: %v to ip set: %s, error: %v", entry, set.Name, err)
+				klog.ErrorS(err, "Failed to add entry to ip set", "entry", entry, "ipSet", set.Name)
```

Comment:
Nit: Update to "Failed to add ip set entry to ip set", makes it consistent with the delete.

Reply:
Make sense, thanks, updated!